src/codecatalyst/activation.ts

```diff
@@ -96,3 +98,29 @@ export async function activate(ctx: ExtContext): Promise<void> {
         authProvider.removeSavedConnection()
     })
 }
+
+async function showReadmeFileOnFirstLoad(workspaceState: vscode.ExtensionContext['workspaceState']): Promise<void> {
+    // Check dev env state to see if this is the first time the user has connected to a dev env
+    const isFirstLoad = workspaceState.get('aws.codecatalyst.devEnv.isFirstLoad', true)
+
+    if (!isFirstLoad) {
+        return
+    }
+
+    const readmeUri = await vscode.workspace.findFiles('README.md').then(files => {
+        if (files.length === 0) {
+            return undefined
+        }
+        return files[0]
+    })
+
+    if (readmeUri === undefined) {
+        getLogger().info('codecatalyst: README.md not found in project root')
+        return
+    }
+
+    // Show rendered readme file to user
+    await vscode.commands.executeCommand('markdown.showPreview', readmeUri)
```

Comment:
Hopefully this works in Cloud9. Else, `markdown.showPreviewToSide` is known to work: https://github.com/aws/aws-toolkit-vscode/blob/61bbc248221ac21e30fa74aaec20a12cae59c10a/src/lambda/commands/createNewSamApp.ts#L94

Reply:
I was able to trigger `markdown.showPreview` from the UI so it should work.